app/util.go

```diff
@@ -15,8 +15,12 @@ import (
 	pocketKeeper "github.com/pokt-network/pocket-core/x/pocketcore/keeper"
 )
 
-func GenerateAAT(appPubKey, clientPubKey string, key crypto.PrivateKey) (aatjson []byte, err error) {
-	aat, er := pocketKeeper.AATGeneration(appPubKey, clientPubKey, key)
+// GenerateAAT generates an AAT to be used for relay servicing.
+// - appPubKey is the public key of the application that's paying for on-chain service.
```

Comment:
This comment is very helpful: a link to a grove doc page that describes the relationship between apps, nodes, and relays, would be great.

Reply:
> a link to a grove doc page that describes the relationship between apps, nodes, and relays, would be great.

I'm unsure what you mean here.

1. Should we link to this file from within the grove documentation?
2. Should we link to a grove doc in the comment?
3. Something else?

Reply:
@adshmh Bump